packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -732,6 +742,131 @@ Link: <https://${yourOktaDomain}/api/v1/groups?after=00g1emaKYZTWRYYRRTSK&limit=
 ]
 ```
 
+#### List Groups with Search
+
+<ApiLifecycle access="ea" />
+
+Searches for groups based on the properties specified in the search parameter
+
+Property names in the search parameter are case sensitive, whereas operators (`eq`, `sw`, etc.) and string values are case insensitive.
+
+This operation:
+
+* Supports [pagination](/docs/reference/api-overview/#pagination).
+* Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding).
+For example, `search=type eq "OKTA_GROUP"` is encoded as `search=type+eq+%22OKTA_GROUP%22`.
+Examples use cURL-style escaping instead of URL encoding to make them easier to read.
+Use an ID lookup for records that you update to ensure your results contain the latest data.
+* Searches many properties:
+   - Any group profile property, including imported app group profile properties.
+   - The top-level properties `id`, `created`, `lastMembershipUpdated`, `lastUpdated`, and `type`.
+   - The <ApiLifecycle access="ea" /> [source](#group-attributes) of groups with type `APP_GROUP`, accessed as `source.id`.
+
+| Search Term Example                                       | Description                                                               |
+| :-------------------------------------------------------- | :------------------------------------------------------------------------ |
+| `type eq "APP_GROUP"`                                     | Groups that have a `type` of `APP_GROUP`                                  |
+| `lastMembershipUpdated gt "yyyy-MM-dd'T'HH:mm:ss.SSSZ"`   | Groups whose memberships were last updated after a specific timestamp     |
+| `id eq "00gak46y5hydV6NdM0g4"`                            | Groups with a specified `id`                                              |
+| `profile.name eq "West Coast Users"`                      | Groups that have a `name` of `West Coast Users`                           |
+| `profile.samAccountName sw "West Coast" `                 | Groups whose `samAccountName` starts with `West Coast`                    |
+| `source.id` eq `0oa2v0el0gP90aqjJ0g7`                     | Groups that have the source application with a specified `source.id`      |
+
+##### Search Examples
+
+List groups with type `APP_GROUP` that were created before `01/01/2014` and have a source application id `0oa2v0el0gP90aqjJ0g7`.
+
+    search=type eq "APP_GROUP" and (created lt "2014-01-01T00:00:00.000Z" and source.id eq "0oa2v0el0gP90aqjJ0g7")
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/groups?search=lastUpdated+gt+\"2015-10-01T00:00:00.000Z\"+or+lastMembershipUpdated+gt+\"2015-10-01T00:00:00.000Z\""
+```
+
+##### Response example
```

Comment:
I don't think the two groups below match the filter above - can you update / run the example against a test org to make sure things are working as expected?

Reply:
The filter under Search Examples uses type eq "APP_GROUP" - was that intended?

Reply:
The example listed under "Search Examples" is actually a different search than the one used in "request example". I tend to follow the same pattern as the existing User search documentation, where the "Search Examples" just gives example search strings and "Request Example" shows a different sample request and response.

Reply:
In that case, do we want to add a few more examples to further highlight that more things are searchable? Maybe:
```
source.id eq "blah"
profile.name eq "blah"
profile.<some AD group profile attribute> eq "blah"
<the existing compound filter from above>
```

Reply:
Added another compound search example -`Search Term Example` table displays simple filter examples and `Search Examples` lists compound search examples.